spec/models/team_spec.rb

```diff
@@ -148,31 +148,24 @@
     end
   end
 
-  describe '#state' do
+  describe '#confirmed?' do
     let(:first_student) { create(:user) }
     let(:second_student) { create(:user) }
     let(:team) { FactoryGirl.create(:team) }
     let(:role_name) { 'student' }
-    it 'returns "pending" when only one student present' do
+    it 'will not be confirmed if only one student present' do
       team.attributes = { roles_attributes: [{ name: role_name, user_id: first_student.id }] }
-      expect { team.save! }.not_to change { team.confirmed? }
```

Comment:
I'd prefer the the `expect { }.to … ` syntax since that puts a object state-change under test. Can you change this to:

```ruby
expect { teanm.save! }.not_to change { team.confirmed? }
```

? Thank you!

(also the example below)